pkg/controller/user/user_controller.go

```diff
@@ -419,6 +464,25 @@ func (c *Controller) updateFederatedUser(fedUser *iamv1alpha2.FederatedUser) err
 	return nil
 }
 
+func (c *Controller) ldapSync(user *iamv1alpha2.User) error {
+	encrypted, _ := strconv.ParseBool(user.Annotations[iamv1alpha2.PasswordEncryptedAnnotation])
+	if encrypted {
```

Comment:
Don't sync user if the password is encrypted, why ?

Reply:
Use encrypted password cannot logged in through LDAP. Jenkins authorization mode `Logged in users can do anything` just lookup authenticated user in LDAP, user password  is not important, because regular users are not allowed to log in to Jenkins directly.